lms/templates/certificates/_about-edx.html

```diff
@@ -0,0 +1,11 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%namespace file='/theme-variables.html' import="get_certificates_settings" />
+<section class="about-item about-edx">
+    <h2 class="about-edx-title hd-4">${company_about_title}</h2>
+
+    <div class="about-edx-copy copy copy-meta">
+        <p>${get_certificates_settings()['footer_about_platform_text']}</p>
+
+        <a class="action" href="${get_certificates_settings()['footer_about_platform_url']}">${company_about_urltext}</a>
```

Comment:
We should hide the link if the value is unset

Reply:
Changed. Pushing soon.